lib/Backends/CPU/CPUBackend.cpp

```diff
@@ -417,6 +417,13 @@ bool CPUBackend::isOpSupported(Kinded::Kind opKind, ElemKind elementTy) const {
   return true;
 }
 
+bool CPUBackend::shouldLower(Node *N) const {
```

Comment:
curious why Node is not const Node *N in the interface

Reply:
just follow the virtual function declared in Backends.h :P

Reply:
question is: why it is not const in the virtual function declared in Backends.h?

Reply:
yeah, I meant interface in Backends.h. NBD, I'll fix those.